orkui/model/model.Login.php

```diff
@@ -12,6 +12,11 @@ function logout($userid) {
 		unset($this->session->user_name);
 		unset($this->session->token);
 		unset($this->session->timeout);
+		unset($this->session->kingdom_id);
+		unset($this->session->kingdom_name);
+		if (isset($_SESSION['is_authorized_mundane_id'])) {
```

Comment:
Even after logout this key was still around and it is used in the authorization code so I'm explicitly clearing it on logout